gulpfile.js

```diff
@@ -20,13 +23,12 @@ var jekyllConfig = jekyllConfigs.LOCAL
 var sources = {
   content: 'app/**/*.{markdown,md,html,txt,yml,yaml}',
   styles: 'app/_assets/stylesheets/**/*.{less,css}',
-  js: 'app/_assets/javascripts/**/*.js',
+  js: ['app/_assets/javascripts/**/*.js', 'bower_components/bootstrap/js/dropdown.js'],
   images: 'app/_assets/images/**/*',
   fonts: 'bower_components/font-awesome/fonts/**/*.*'
 }
 
 gulp.task('styles', function () {
-  var glob = require('glob')
```

Comment:
not to fit my sauce, it looked weird, like it was done after original changes (like `connect` below) so I thought it was a mistake ... easily reverted.